go/tasks/pluginmachinery/bundle/fail_fast.go

```diff
@@ -0,0 +1,59 @@
+package bundle
+
+import (
+	"context"
+	"time"
+
+	pluginMachinery "github.com/lyft/flyteplugins/go/tasks/pluginmachinery"
+
+	"github.com/lyft/flyteplugins/go/tasks/pluginmachinery/core"
+)
+
+const failFastExecutorName = "fail-fast"
+
+type failFastHandler struct{}
+
+func (h failFastHandler) GetID() string {
+	return failFastExecutorName
+}
+
+func (h failFastHandler) GetProperties() core.PluginProperties {
+	return core.PluginProperties{}
+}
+
+func (h failFastHandler) Handle(_ context.Context, _ core.TaskExecutionContext) (core.Transition, error) {
+	occuredAt := time.Now()
+	return core.DoTransition(core.PhaseInfoFailure("AlwaysFail",
```

Comment:
to the message can we add the task_type?

Reply:
Also other information like TaskID?

Reply:
I don't see that exposed in the [TaskExecutionContext](https://github.com/lyft/flyteplugins/blob/master/go/tasks/pluginmachinery/core/exec_metadata.go#L24) @kumare3 - should i add it or am I looking in the wrong place?

Reply:
https://github.com/lyft/flyteplugins/blob/master/go/tasks/pluginmachinery/core/exec_metadata.go#L28